desktop/start.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
```

Comment:
heh, I didn't know you could write node scripts with a shebang like this.

Reply:
Ah. It only works for Unix. But we really don't need this line so I will remove it.